erpnext/manufacturing/doctype/workstation/test_workstation.py

```diff
@@ -34,9 +98,12 @@ def make_workstation(*args, **kwargs):
 			"doctype": "Workstation",
 			"workstation_name": workstation_name
 		})
-
+		doc.hour_rate_rent = args.hour_rate_rent
+		doc.hour_rate_labour = args.hour_rate_labour
```

Comment:
```suggestion
		doc.hour_rate_rent = args.get("hour_rate_rent")
		doc.hour_rate_labour = args.get("hour_rate_labour")
```

Not mandatory fields might cause unnecessary test failures.